cirq/contrib/qasm_import/_parser_test.py

```diff
@@ -368,39 +392,102 @@ def test_unknown_function():
         parser.parse(qasm)
 
 
-def test_u_gate_zero_params_error():
+rotation_gates = [
+    ('rx', cirq.Rx),
```

Comment:
Are there any tests actually using these statements?

Reply:
Yeah all three new tests are parameterized and using these